pyright_reportGeneralTypeIssues.json

```diff
@@ -1,6 +1,8 @@
 {
     "typeCheckingMode": "off",
+    "pythonVersion": "3.11",
```

Comment:
Just want to make sure this is necessary - if this was removed it won't be taken from pyproject.toml?

Reply:
The CI runs pyright twice: without reportGeneralTypeIssues on all files and only with reportGeneralTypeIssues on a subset. To achieve that we need two configuration files. I put this line here for consistency, but we might actually not have any errors if we were to remove it. Will test it.

Reply:
I removed this line